core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -621,6 +622,7 @@ private void printHelp(OptionParser parser) throws IOException {
                 + "reset tool!\n\n"
         );
         parser.printHelpOn(System.err);
+        Exit.exit(1);
```

Comment:
We should not exit hard here, but return a appropriate error code if necessary, and propagate it back. Only `main()` should use `exit` to return error code to OS. Using `exit() `makes it hard to test code.